app/pathogen/arbovirus/dashboard/(visualizations)/median-seroprevalence-by-who-region-and-age-group-table.tsx

```diff
@@ -97,13 +143,44 @@ export const MedianSeroprevalenceByWhoRegionAndAgeGroupTable = () => {
     [currentlySelectedArbovirus, tableDatasets]
   );
 
+  const downloadCsv = useCallback(() => {
+    const dataForCsv = typedObjectEntries(tableDatasets).flatMap(
+      ([arbovirus, dataForArbovirus]) =>
+        typedObjectEntries(dataForArbovirus).flatMap(
+          ([whoRegion, dataForWhoRegion]) => ({
```

Comment:
So arboviruses and who regions that result in N/As across the board aren't included as visible from the filtered file I attached in the PR description.